src/xcode/ENA/ENA/Source/Scenes/App/AppNavigationController.swift

```diff
@@ -8,17 +8,25 @@ import UIKit
 class AppNavigationController: UINavigationController {
 	private var scrollViewObserver: NSKeyValueObservation?
 	
-	@available(iOS 13.0, *)
-	private(set) lazy var defaultScrollEdgeAppearance: UINavigationBarAppearance? = nil
+	private(set) lazy var _defaultScrollEdgeAppearance: Any? = nil
 
+	@available(iOS 13.0, *)
+	fileprivate var defaultScrollEdgeAppearance: UINavigationBarAppearance {
+		if _defaultScrollEdgeAppearance == nil {
+			_defaultScrollEdgeAppearance = UINavigationBarAppearance()
+		}
+		// swiftlint:disable:next force_cast
+		return _defaultScrollEdgeAppearance as! UINavigationBarAppearance
+	}
```

Comment:
I completely agree with that, cause I see no other solution at the moment. Would be nice, when there is a better solution without `Any` and `Force Cast` next time ...